lib/src/asn1/liblte_mme.cc

```diff
@@ -3019,6 +3020,10 @@ LIBLTE_ERROR_ENUM liblte_mme_unpack_emergency_number_list_ie(uint8**
     emerg_num_list->N_emerg_nums = 0;
     while (length < sent_length) {
       idx                                               = emerg_num_list->N_emerg_nums;
+      if(idx >= LIBLTE_MME_EMERGENCY_NUMBER_LIST_MAX_SIZE) {
+        srsran::console("Error: The length of emergency number list is too long!\n");
+        return (err);
```

Comment:
use `return err;` like everywhere else

Reply:
OK, but I found many places use ```(err)`` rather than  ```err``.

Reply:
I am sorry, I just checked the code right below and it was using `return err;`. But as you pointed out the usage in the file is rather mixed. I didn't mean to be a nitpicking person.